sfge/src/main/java/com/salesforce/rules/UnusedMethodRule.java

```diff
@@ -45,7 +67,367 @@ protected boolean isEnabled() {
     @Override
     protected List<Violation> _run(
             GraphTraversalSource g, GraphTraversal<Vertex, Vertex> eligibleVertices) {
-        return new ArrayList<>();
+        reset(g);
+        List<MethodVertex> candidateVertices = getCandidateVertices(eligibleVertices);
+        seekMethodUsages(candidateVertices);
+        return convertMethodsToViolations();
+    }
+
+    private void reset(GraphTraversalSource g) {
+        this.g = g;
+        this.unusedMethods.clear();
+        this.internalMethodCallsByDefiningType.clear();
+    }
+
+    private List<MethodVertex> getCandidateVertices(
+            GraphTraversal<Vertex, Vertex> eligibleVertices) {
+        return SFVertexFactory.loadVertices(
+                g, eligibleVertices.hasLabel(NodeType.METHOD).hasNot(Schema.IS_STANDARD));
+    }
+
+    private void seekMethodUsages(List<MethodVertex> candidateVertices) {
+        for (MethodVertex candidateVertex : candidateVertices) {
+            // If the method is one that is ineligible to be analyzed, skip it.
+            if (methodIsIneligible(candidateVertex)) {
+                continue;
```

Comment:
Would be helpful to have a log line with information on why we decided not to pursue a method.